src/components/common/YourCompany/EntityName.vue

```diff
@@ -367,14 +370,21 @@ export default class EntityName extends Mixins(NameRequestMixin) {
     return (this.correctionNameChoices.length > 0)
   }
 
-  /** The Name Request applicant info. */
-  get nrApplicant (): NameRequestApplicantIF {
-    return this.getNameRequest?.applicant
+  get nrFullName (): string {
+    return this.formatFullName(this.getNameRequest.applicants)
```

Comment:
Full Name and Full Address (below) are no longer stored in the namerequest applicant object.